snooty.toml

```diff
@@ -70,6 +70,7 @@ client-sdk = "Realm SDK"
 client-sdks = "Realm SDKs"
 compass = "MongoDB Compass"
 compass-short = "Compass"
+data-api = "Data API"
```

Comment:
why are we using source constant for this?

Reply:
Atlas uses the source constant so for easy copy-pasta I added it here

Reply:
I actually went through and de-Data API'd the examples to match the rest of the app services docs, so we don't really need this anymore. Removed.